heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerRunner.java

```diff
@@ -48,13 +48,18 @@
   private final Config runtime;
   private final Command command;
   private final ISchedulerClient schedulerClient;
+  private final boolean running;
 
-  public RuntimeManagerRunner(Config config, Config runtime,
-                              Command command, ISchedulerClient schedulerClient) {
+  public RuntimeManagerRunner(Config config,
+                              Config runtime,
+                              Command command,
+                              ISchedulerClient schedulerClient,
+                              boolean running) {
```

Comment:
2 minor comments:
1. Why pass the flag `running` in the constructor, rather than in the `Config runtime`?
2. Name the boolean as `isRunning` will be more expressive and more java style.

Reply:
Thanks!

For 1, "config" is for the topology configuration. running flag is not a config.
For 2, will do.

Reply:
OK Fair enough.